README.md

```diff
@@ -63,8 +63,15 @@ This section describes the optional enterprise functionality that can be enabled
 * `OIDC_CONFIG`
   *  Decription: Enable OpenIdConnect Client configuration to be read from environment variables.  
   *  XML Snippet Location: [oidc-config.xml](/common/helpers/build/configuration_snippets/oidc-config.xml)
-  *  Note: The following variables will be read:  OIDC_CLIENT_ID, OIDC_CLIENT_SECRET, OIDC_DISCOVERY_URL.  
-
+  *  Note: The following variables will be read:  OIDC_CLIENT_ID, OIDC_CLIENT_SECRET, OIDC_DISCOVERY_URL.
+* `KEYSTORE_REQUIRED`
+  *  Description: Enables keystore element, which holds the key that is required to make an SSL connection.
+  *  XML Snippet Location: [keystore.xml](/common/helpers/build/configuration_snippets/keystore.xml)
+  *  Note: Keystore password and security certificate will be generated if users do not provide their own. The user provided keystore.xml (with defined password) and certificate can be applied using the following commands inside of their Dockerfile:
```

Comment:
@Rachelwrq It'll be good to address the users directly, since the Readme is for end-users. Suggest following change:
`will be generated if you do not provide your own. You can provide keystore.xml (with defined password) and certificate using the following instructions inside your Dockerfile:`